docs/app.md

```diff
@@ -11,7 +11,7 @@ Manage apps, and app installations in your projects
 * [`mw app dependency list`](#mw-app-dependency-list)
 * [`mw app dependency update [INSTALLATION-ID]`](#mw-app-dependency-update-installation-id)
 * [`mw app dependency versions SYSTEMSOFTWARE`](#mw-app-dependency-versions-systemsoftware)
-* [`mw app download [INSTALLATION-ID]`](#mw-app-download-installation-id)
+* [`mw app download [INSTALLATION-ID] TARGET`](#mw-app-download-installation-id-target)
```

Comment:
Ah; now I remember why we didn't do this in the first place. This is not valid, because `[INSTALLATION-ID]` is optional any may not be followed by a required argument. So the only valid order would be `TARGET [INSTALLATION-ID]`, which also seems unintuitive. I'll just close this and leave this as-is.